gap-analysis/index.md

```diff
@@ -345,47 +345,11 @@ The <semantics> element may be used to attach content markup as an annotation to
 </semantics>
 ```
 
+The `<csymbol>point<csymbol/>` element in the above example shows one style of markup that can be used to refer to operators that are not part of standard content MathML. It is used here to illustrate the style of parallel markup elements that could be used to represent the point example introduced above.  Other strategies to handle references to symbols outside of MathML include URI-style attributes (`definitionURL`), and attributes that link into OpenMath content dictionaries (`cdbase` and `cd`).
```

Comment:
1. Thanks to David's comments, I can also say, the markup above should be  `</csymbol>` not `<csymbol/>`.

2. There is no "standard" content MathML, as far as I'm ware? Do you mean pragmatic? Is this point important? Safer to erase ` that are not part of standard content MathML`.

3. `symbols outside of MathML` is hard for me to wrap my mind around, unless `MathML` now is an abbreviation for `pragmatic Content MathML`.

Reply:
I believe this is now resolved as per the discussion in the paragraph.

Because people can now link to this document and we are *maybe* down to discussing one sentence, I'm going to do the commit as this is a big clean up that needs to be visible.